partiql-logical-planner/src/builtins.rs

```diff
@@ -725,6 +726,42 @@ fn function_call_def_coll_every() -> CallDef {
     }
 }
 
+fn vararg_fn_overloads<F>(f: F) -> Vec<CallSpec>
+where
+    F: Clone + 'static + Fn(Vec<ValueExpr>) -> logical::ValueExpr + Send + Sync,
+{
+    (1..=FN_VAR_ARG_MAX)
+        .map(|n| CallSpec {
+            input: std::iter::repeat(CallSpecArg::Positional).take(n).collect(),
+            output: Box::new(f.clone()),
+        })
+        .collect()
+}
+
+fn function_call_def_tupleunion() -> CallDef {
+    CallDef {
+        names: vec!["tupleunion", "tuple_union"],
```

Comment:
For this and the merge operation, the PartiQL Spec only refers to `TUPLEUNION` -- not with snake case. Is there a requirement to add this to the language?